cmd/minishift/cmd/root.go

```diff
@@ -52,10 +52,18 @@ import (
 
 const (
 	showLibmachineLogs    = "show-libmachine-logs"
+	profileCmd            = "profile"
 	profileFlag           = "profile"
+	profileSetCmd         = "set"
 	enableExperimentalEnv = "MINISHIFT_ENABLE_EXPERIMENTAL"
+	invalidProfileName    = "Profile names must consist of alphanumeric characters only."
 )
 
+var profileCmdAlias = []string{
+	"profiles",
+	"instance",
+}
```

Comment:
I think we should move this in to initializeProfile() as it is only used in it and we do not need these as global variables